src/hffs/fs.py

```diff
@@ -182,40 +206,41 @@ def _open(
         **kwargs,
     ):
         if mode == "ab":
-            raise NotImplementedError("Appending to remote files is not supported")
+            raise NotImplementedError("Appending to remote files is not yet supported.")
         path = self._strip_protocol(path)
         return HfFile(self, path, mode=mode, **kwargs)
 
-    def _rm(self, path):
+    def _rm(self, path, **kwargs):
         path = self._strip_protocol(path)
         operations = [huggingface_hub.CommitOperationDelete(path_in_repo=path)]
-        commit_message = f"Delete {path} with hffs"
+        commit_message = f"Delete {path}"
         self._api.create_commit(
             repo_id=self.repo_id,
             repo_type=self.repo_type,
             token=self.token,
             operations=operations,
             revision=self.revision,
-            commit_message=commit_message,
+            commit_message=kwargs.get("commit_message", commit_message),
+            commit_description=kwargs.get("commit_description", None),
         )
         self.invalidate_cache()
 
-    def rm(self, path, recursive=False, maxdepth=None):
+    def rm(self, path, recursive=False, maxdepth=None, **kwargs):
         paths = self.expand_path(path, recursive=recursive, maxdepth=maxdepth)
         file_paths = [path for path in paths if not self.isdir(path)]
         operations = [huggingface_hub.CommitOperationDelete(path_in_repo=file_path) for file_path in file_paths]
         commit_message = f"Delete {path} "
         commit_message += "recursively " if recursive else ""
         commit_message += f"up to depth {maxdepth} " if maxdepth is not None else ""
-        commit_message += "with hffs"
         # TODO: use `commit_description` to list all the deleted paths?
         self._api.create_commit(
             repo_id=self.repo_id,
             repo_type=self.repo_type,
             token=self.token,
             operations=operations,
             revision=self.revision,
-            commit_message=commit_message,
+            commit_message=kwargs.get("commit_message", commit_message),
+            commit_description=kwargs.get("commit_description", None),
```

Comment:
```suggestion
            commit_description=kwargs.get("commit_description"),
```

`None` could be implicit here